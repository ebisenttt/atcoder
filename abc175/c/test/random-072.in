-816704192199011 520679508002519 677669605042813
