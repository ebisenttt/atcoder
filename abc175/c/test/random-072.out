139034587156198
